Source/WarpX.H

```diff
@@ -120,6 +121,12 @@ public:
 
     void Evolve (int numsteps = -1);
 
+    void EvolveImplicitPicardInit (int lev);
+    void SaveParticlesAtStepStart (WarpXParticleContainer& pc, int lev);
```

Comment:
Could we change the function name to make it clear that this is used only in the implicit algorithm:
```suggestion
    void SaveParticlesAtImplicitStepStart (WarpXParticleContainer& pc, int lev);
```